flake.nix

```diff
@@ -64,6 +64,8 @@
       dell-xps-15-9560 = import ./dell/xps/15-9560;
       dell-xps-15-9560-intel = import ./dell/xps/15-9560/intel;
       dell-xps-15-9560-nvidia = import ./dell/xps/15-9560/nvidia;
+      dell-xps-15-9570-intel = import ./dell/xps/15-9570/intel;
+      dell-xps-15-9570-nvidia = import ./dell/xps/15-9570/nvidia;
```

Comment:
Do we also need to expose `./dell/xps/15-9570` as an option?

Reply:
I think having intel or nvidia should be fine. People have to make a choice between the two, no?

Reply:
I agree with you. On these laptops, NVIDIA consumption is quite high. To prevent battery drain, I have added these options